package.json

```diff
@@ -50,5 +51,8 @@
   },
   "engines": {
     "npm": ">=7.0.0"
+  },
+  "dependencies": {
+    "vue-tsc": "^2.0.29"
```

Comment:
looks like the error you're getting in tryorama in related to vue-tsc version? I think this should only be a dependency in the `ui/package.json` -- so we don't need it here (also looks like they are on conflicting versions).